system/languages/nl.php

```diff
@@ -50,6 +50,7 @@
 		"default-page" => "standaard pagina",
 		"default-language" => "standaard taal",
 		"digest-interval" => "notificatie-interval",
+		"default-new-page-type" => "standaard type voor nieuwe pagina",
```

Comment:
`standaardtype` is one word, I think :-)

Reply:
Also, I would say "pagina's" rather than "pagina".

Reply:
Fixed